sleap_nn/config/data_config.py

```
@@ -6,7 +6,8 @@
 
 from attrs import define, field, validators
 from omegaconf import MISSING
-from typing import Optional, Tuple, Any, Union, List
+from typing import Optional, Tuple, Any, List
```

Ruff (0.8.2)

`typing.Any` imported but unused

Remove unused import: `typing.Any`

(F401)